scripts/coverage_report.sh

```diff
@@ -69,28 +69,14 @@ then
 	fi
 fi
 
+# Set the flags necessary for coverage output
 export RUSTFLAGS="-Zprofile -Ccodegen-units=1 -Cinline-threshold=0 -Coverflow-checks=off -Zno-landing-pads"
 export CARGO_INCREMENTAL=0
 
 # Clean the project
 echo "Cleaning project..."
 (cd $TEST_DIR; cargo clean)
 
-# Build with flags necessary for coverage output
```

Comment:
nit: this comment should be preserved for all to grasp the crucial use of RUSTFLAGS above.

Reply:
I had moved that comment (with a minor rephrasing) right above the setting of RUSTFLAGS. Are you asking for something more than that?